src/chocolatey.resources/helpers/functions/Get-WebHeaders.ps1

```diff
@@ -62,7 +68,10 @@ param(
 
   $request = [System.Net.HttpWebRequest]::Create($url);
   $defaultCreds = [System.Net.CredentialCache]::DefaultCredentials
-  if ($defaultCreds -ne $null) {
+  if ($credential -ne $null) {
+    Write-Host "Using credential."

```

Comment:
This should be either Write-Verbose or Write-Debug. And it should provide more information. What credential? User passed or something default?